specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/ResourceProvider.json

```diff
@@ -1605,6 +1697,23 @@
           "x-ms-client-flatten": true
         }
       }
+    },
+    "VirtualNetworkIntegrationRequest": {
+      "description": "Virtual Network Integration request content.",
+      "required": [
+        "subnetResourceId"
+      ],
+      "type": "object",
+      "properties": {
+        "subnetResourceId": {
+          "description": "The ARM Resource ID of the subnet",
+          "type": "string"
+        }
+      }
+    },
+    "PurgedUnusedVirtualNetworkSuccessResponse": {
```

Comment:
>PurgedUnusedVirtualNetworkSuccessResponse [](http://example.com/codeflow?start=5&length=41)

If this property takes a fixed set of values, please model it as a string enum, rather than open-ended string.

Reply:
Based on the enum values proposed, since this seems to be an output message, maybe string was better?

Reply:
Or you could change the values to something like 
-PurgedUnusedVNets
-NoUnusedVnetsToPurge